spec/support/shared_examples/create_product_with_price_rule_shared_examples.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples "creates a product with billing mode" do |product_type, billing_mode|
```

Comment:
I think it if it's used by a single example group we can define these shared examples with the same file

Reply:
It's used in 2 different files:

spec/system/admin/creating_a_service_spec.rb
spec/system/admin/creating_an_item_spec.rb

Reply:
I hadn't noticed it